ui/css/module/_footnotes.scss

```diff
@@ -5,13 +5,77 @@
   padding: 2px 4px;
   font-size: 0.6em;
   margin-left: .3em;
+  font-style: normal;
 
   &.active {
-    background: $color-cool-blue;
+    border-color: $color-primary-darker;
+    background: $color-primary-darker;
     color: $color-white;
   }
 }
 
+.node-section {
```

Comment:
This document has sections throughout, but I don't think it'd be wise to use that as a differentiator between inline footnotes and others. What do you think about adding a new class for inline footnotes instead?

Relatedly, we may need to use a separate component; I _think_ that right now we're including multiple elements with the same `id` due to the component reuse.